solr/core/src/test/org/apache/solr/update/processor/TolerantUpdateProcessorTest.java

```diff
@@ -260,8 +259,7 @@ public void testMaxErrors0() {
                     "tolerant-chain-max-errors-10", smallBatch, requestParams, "1"));
     assertTrue(
         e.getMessage()
-            .contains(
-                "ERROR: [doc=1] Error adding field 'weight'='b' msg=For input string: \"b\""));
```

Comment:
In `Isn't `For input string: \"b\""` - "b" is still the value? At least it looks like it was in `'weight'='b'`

Reply:
Yea, you're right, still leaks the value from the exception message. The "For input string" text is from `NumberFormatException`, probably in `Integer.parseInt()` when some class somewhere parses a weight field. Question is whether we could instead print something like
> Error adding fidle 'weight'. Exception: NumberFormatException